website/content/docs/commands/job/run.mdx

```diff
@@ -101,10 +101,16 @@ that volume.
   a Consul token must be supplied with appropriate service and kv Consul ACL policy
   permissions.
 
-- `-vault-token`: If set, the passed Vault token is stored in the job before
-  sending to the Nomad servers. This allows passing the Vault token without
+- `-vault-token`: Used to validate if the user submitting the job has
+  permission to run the job according to its Vault policies.
+  A Vault token must be supplied if `allow_unauthenticated = false` is
```

Comment:
Maybe add a link to the `allow_unauthenticated` config page.

Reply:
added link, and had to modify another link since consul also has an `allow_unauthenticated` that's referenced right above here